### PR TITLE
ADD: Visual feedback when changing lightning settings.

### PR DESCRIPTION
**Pull request by marcosrdz**
FIX: Don't change existing wallet's URI. This will cause errors. Only apply to new wallets.
ADD: Lightning wallet details shows where the wallet is connected to

### CODE REVIEW

File: screen/wallets/details.js [hunk 1]
```diff
@@ -120,6 +120,16 @@ export default class WalletDetails extends Component {
                 {loc.wallets.details.type.toLowerCase()}
               </Text>
               <Text style={{ color: '#81868e', fontWeight: '500', fontSize: 14 }}>{this.state.wallet.typeReadable}</Text>
+              {this.state.wallet.type === LightningCustodianWallet.type && (
+                <React.Fragment>
+                  <Text style={{ color: '#0c2550', fontWeight: '500', fontSize: 14, marginVertical: 12 }}>{'connected to'}</Text>
+                  <Text style={{ color: '#81868e', fontWeight: '500', fontSize: 14 }}>
+                    {this.state.wallet.getBaseURI() === LightningCustodianWallet.defaultBaseUri
+                      ? 'BlueWallet LNDHub'
```
Comment (Overtorment) [inline screen/wallets/details.js]: I like it

File: screen/settings/lightningSettings.js [hunk 2]
```diff
@@ -32,25 +35,27 @@ export default class LightningSettings extends Component {
...
-        console.log('inited', w.baseURI);
+  save = () => {
+    this.setState({ isLoading: true }, async () => {
+      this.state.URI = this.state.URI ? this.state.URI : '';
+      if (this.state.URI.trim().length <= 0) {
+        await AsyncStorage.setItem(AppStorage.LNDHUB, '');
+        alert('Your changes have been saved successfully');
+      } else {
+        try {
+          await frisbee.get(this.state.URI).then(console.log);
```
Comment (Overtorment) [inline screen/settings/lightningSettings.js]: validation. nice.

File: screen/settings/lightningSettings.js [hunk 3]
```diff
@@ -32,25 +35,27 @@ export default class LightningSettings extends Component {
     });
   }
 
-  async save() {
-    this.state.URI = this.state.URI ? this.state.URI : '';
-    await AsyncStorage.setItem(AppStorage.LNDHUB, this.state.URI);
-
-    // set each lnd wallets and re-init api
-    for (/** @type {LightningCustodianWallet} */ let w of BlueApp.getWallets()) {
```
Comment (Overtorment) [inline screen/settings/lightningSettings.js]: got concerns

**Comment by Overtorment** [comment]
Ok now Im not sure.
`this.baseURI` is the object property, so it should save/restore with our regular mechanism of saving and loading from storage. But does it have priority over App-wide lndhub base uri..?
Pls investigate

**Comment by Overtorment** [comment]
including domain in `secret` might also be a good idea, for super-portability of wallet backups

**Comment by marcosrdz** [comment]
Please review.

**Comment by Overtorment** [comment]
@marcosrdz  ^^^

**Comment by marcosrdz** [comment]
> @marcosrdz ^^^

Fixed. Please review

**Comment by Overtorment** [comment]
@marcosrdz circleci still fails

**Comment by Overtorment** [comment]
damn, again a bunch of changes in a single branch

**Comment by Overtorment** [comment]
Pls split in separate branches

**Pull request closed without merging**